datahub-web-react/src/app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/SelectSourceStep.tsx

```diff
@@ -1,24 +1,155 @@
-import React, { useEffect } from 'react';
+import { Badge, Icon, colors } from '@components';
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
 
 import { SourceBuilderState } from '@app/ingestV2/source/builder/types';
+import ShowAllCard from '@app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/ShowAllCard';
+import SourcePlatformCard from '@app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/SourcePlatformCard';
+import sourcesJson from '@app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/sources.json';
+import { useCardsPerRow } from '@app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/useCardsPerRow';
+import {
+    CARD_WIDTH,
+    computeRows,
+    groupByCategory,
+    sortByPopularFirst,
+} from '@app/ingestV2/source/multiStepBuilder/steps/step1SelectSource/utils';
 import { IngestionSourceFormStep } from '@app/ingestV2/source/multiStepBuilder/types';
 import { useMultiStepContext } from '@app/sharedV2/forms/multiStepForm/MultiStepFormContext';
 
+const StepContainer = styled.div`
+    padding: 0 20px 20px 20px;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+`;
+
+const Section = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+`;
+
+const SectionHeader = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    font-size: 16px;
+    font-weight: 700;
+    color: ${colors.gray[600]};
+`;
+
+const CardsWrapper = styled.div`
+    display: grid;
+    grid-template-columns: repeat(auto-fit, ${CARD_WIDTH}px);
+    justify-content: start;
+    gap: 8px;
+`;
+
+const LeftSection = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 8px;
+`;
+
+const RightSection = styled.div`
+    display: flex;
+    cursor: pointer;
+`;
+
 export function SelectSourceStep() {
```

Comment:
nice! one thing we'll have to build after this PR is going to be the ability to search/filter for a source by name